### PR TITLE
Include last commit's SHA in .deb package version

### PR DESCRIPTION
**Pull request by azawlocki**
This PR modifies the workflow file for the `Build .deb` action and the dockerfiles that are used to build .deb packages, in order to include the SHA of the current commit in the `Version` property of the .deb packages. After this change, the version property of `yagna.deb` will be something like `0.3.0-8832501e` instead of just `0.3.0`.

This will allow us to identify which version of `yagna` code is actually being tested in integration tests.

### CODE REVIEW

File: .github/workflows/build-deb.yml [hunk 1]
```diff
@@ -16,14 +16,23 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Compute deb package version
+        id: vars
+        run: |
```
Comment (tworec) [inline .github/workflows/build-deb.yml:21]: there is nice syntax faciliating setting env var like this. Also, consider renaming `SHA` into sth more meaningful like
```suggestion
        env:
          COMMIT_SHA: ${{ github.sha }}
        run: |
```

**Comment by tworec** [comment]
there is also nice crate : https://crates.io/crates/git-version

File: .github/workflows/build-deb.yml (hunk 1 above)
Comment (azawlocki) [reply .github/workflows/build-deb.yml:25]: Thanks for comments, I've included your suggestions in my latest comment (https://github.com/golemfactory/yagna/commit/3eccf0ae1a5c938fd1bb20d949b4bad8869038af)

**Comment by azawlocki** [comment]
As for adding commit hash to version reported by `yagna` binary, I'll do this in a separate PR

File: build-deb-router.Dockerfile [hunk 2]
```diff
@@ -1,3 +1,4 @@
 FROM build-deb
+ARG DEB_VERSION=unknown
```
Comment (tworec) [inline build-deb-router.Dockerfile:2]: 👍

**Comment by tworec** [comment]
I forgot to add, that we'd rather not commit such "well-grained" commits individually, but rather as a squash... especially those for moving branch activator back-and-forward ;)

**Comment by azawlocki** [comment]
> @azawlocki I forgot to add, that we'd rather not merge such "well-grained" commits individually to master, but rather as a squash commit... especially those moving branch activator back-and-forward ;)

Got it!

**Pull request merged**